src/Shared/EnvironmentUtilities.cs

```diff
@@ -10,52 +10,12 @@ internal static partial class EnvironmentUtilities
     {
         public static bool Is64BitProcess => Marshal.SizeOf<IntPtr>() == 8;
 
-        //  Copied from .NET Framework source for Environment.Is64BitOperatingSystem (along with Win32Native APIs)
-        public static bool Is64BitOperatingSystem
```

Comment:
This was interesting because it failed my unit test (returned false). No idea why, but I think they've since added `RuntimeInformation.OSArchitecture` so I don't think there's any reason for this.